src/main/java/io/github/ryotan/code/Code.java

```diff
@@ -0,0 +1,255 @@
+/*
+ * These codes are licensed under CC0.
+ *
+ * http://creativecommons.org/publicdomain/zero/1.0/deed
+ * http://creativecommons.org/publicdomain/zero/1.0/deed.ja
+ */
+
+package io.github.ryotan.code;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import io.github.ryotan.code.CodeEnum.AliasLabel;
+import io.github.ryotan.code.CodeEnum.Filters;
+import io.github.ryotan.code.CodeEnum.ShortLabel;
+import io.github.ryotan.code.util.CodeEnumReflectionUtil;
+
+/**
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [First sentence of Javadoc is incomplete (period is missing) or not present.](https://www.codacy.com/app/ryotan/code-enum-support/file/2948699296/issues/source?bid=3375378&fileBranchId=3375797#l21)